.scrutinizer.yml

```diff
@@ -6,7 +6,7 @@ filter:
     paths: ["pyof/*", "tests/*"]
 build:
     environment:
-        python: 3.9.12
+        python: 3.11.8
```

Comment:
This `.scrutinizer.yml` we can maintain it since it used to be here. But, please map an issue in the backlog, I think it could get removed too and re-use the general one as mentioned on https://github.com/kytos-ng/maintenance/pull/87/files#r1526311292

Reply:
Deleted